config/initializers/assets.rb

```diff
@@ -7,5 +7,3 @@
 
 # Add additional assets to the asset load path.
 # Rails.application.config.assets.paths << Emoji.images_path
-# Add Yarn node_modules folder to the asset load path.
-Rails.application.config.assets.paths << Rails.root.join('node_modules')
```

Comment:
I'm assuming we need this but if testing a deployed environment shows an environment with working JS functions, fine to dump it.